articles/azure-arc/kubernetes/cluster-connect.md

```diff
@@ -179,8 +179,13 @@ A conceptual overview of this feature is available in [Cluster connect - Azure A
     ```
 
     ```console
-    $TOKEN=(kubectl get secret demo-user-secret -o jsonpath='{$.data.token}' | base64 -d | sed $'s/$/\\\n/g')
+    TOKEN=$(kubectl get secret demo-user-secret -o jsonpath='{$.data.token}' | base64 -d | sed $'s/$/\\\n/g')
     ```
+1. Get the token to output to console
+  
+  ```console
+  echo $TOKEN
+  ```
```

Comment:
```suggestion
1. Get the token to output to console
  
     ```console
     echo $TOKEN
     ```
```
indenting code block to render under the step number